### ISSUE TITLE
Button color styles do not follow theme styling

### ISSUE THREAD

**Issue by 3coins** [jupyterlab/jupyterlab#11260]
## Description
Button styling is inconsistently applied within the Jupyter notebooks. e.g., inside the kernel switcher dialog, the **Submit** and **Cancel** buttons have hardcoded colors that do not change when a new theme is installed. In addition, the material colors selected for normal (500) and hover (600) states are not accessible with a white font color.

*Notice the difference in background colors for the toolbar button at the top left and the primary button in the notebook.*

<img width="796" alt="buttons-theme-monokai" src="https://user-images.githubusercontent.com/289369/136457365-92b46296-03df-4596-86dd-60e2c1675654.png">

## Reproduce

1. Install a new theme
```bash
jupyter labextension install @arbennett/base16-monokai
```
2. Open a new notebook, and add this in a markdown cell
```html
<button class="jp-mod-accept jp-mod-styled">Button (Accept)</button><br/><br/>
<button class="jp-mod-warn jp-mod-styled">Button (Warn)</button><br/><br/>
<button class="jp-mod-reject jp-mod-styled">Button (Reject)</button>
```
3. Change theme to monokai, **Settings** > **Theme** > **base16-monokai**

## Context

- Operating System and version: All
- Browser and version: All
- JupyterLab version: All

<details><summary>Troubleshoot Output</summary>
<pre>
Paste the output from running `jupyter troubleshoot` from the command line here.
You may want to sanitize the paths in the output.
</pre>
</details>

<details><summary>Command Line Output</summary>
<pre>
Paste the output from your command line running `jupyter lab` here, use `--debug` if possible.
</pre>
</details>

<details><summary>Browser Output</summary>

<pre>
Paste the output from your browser Javascript console here, if applicable.

</pre>
</details>
